x/gov/client/testutil/deposits.go

```diff
@@ -73,12 +74,15 @@ func (s *DepositTestSuite) createProposal(val *network.Validator, initialDeposit
 		exactArgs = append(exactArgs, fmt.Sprintf("--%s=%s", cli.FlagDeposit, initialDeposit.String()))
 	}
 
-	_, err := MsgSubmitProposal(
+	govAcc, err := s.cfg.AccountRetriever.GetAccount(val.ClientCtx, authtypes.NewModuleAddress(types.ModuleName))
+	s.Require().NoError(err)
+
+	proposal := []sdk.Msg{types.NewMsgVote(govAcc.GetAddress(), 1, types.OptionYes)}
```

Comment:
am i reading it correctly that the proposal msg that is getting voted on is itself a vote proposal?

Reply:
yes lol. This is just a dummy proposal used in testing but essentially the governance account would vote on a proposal.